django/contrib/auth/password_validation.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import unicode_literals
+
+import os
+import re
+from difflib import SequenceMatcher
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured, ValidationError
+from django.template.defaultfilters import unordered_list
+from django.utils.encoding import force_text
+from django.utils.module_loading import import_string
+from django.utils.six import string_types
+from django.utils.translation import ugettext as _
+
+
+def get_password_validators():
```

Comment:
I'm theoretically reluctant to this being a global module rather than something tied to the User model.

In practice AUTH_USER_MODEL is already a global singleton so this doesn't change the picture.

(Sorry if I brough this up before.)